nix/migration-tests.nix

```diff
@@ -56,6 +56,8 @@ let
       sha256 = "1nfzfdv03nz2bhin2jflxsmh597vkjifb76jsdg0384x89gi0a10"; }
     { rev = "v2020-09-30";
       sha256 = "19vmyq0m67ih295z1y8ng2rdn85mi0rvw26k711lbxsqri7fmyvz"; }
+    { rev = "v2020-10-13";
```

Comment:
Initially I thought the list is up-to-date... But ofc we need current release too :facepalm: